smoke.sh

```diff
@@ -68,6 +72,8 @@ function finish ()
     if [ $RET -ne 0 ]; then
         cat /build/dbench-logs || true
     fi
+    #Move statedumps to be archived
+    mv /var/run/gluster/*dump* /var/log/glusterfs/
```

Comment:
We just want the statedumps to be collected as part of the tarball. Not sure I understand the intention of making it a mount. Could you explain more?